forch/faucetizer.py

```diff
@@ -18,6 +18,10 @@
 
 INCLUDE_FILE_SUFFIX = '_augmented'
 TESTING_PORT_IDENTIFIER_DEFAULT = 'TESTING'
+BEHAVIOR = 'behavior'
```

Comment:
these are overly generic for top-level constants, so should be qualified somehow... e.g. so it reads DEVICE.STATIC or DEVICE_STATIC

Reply:
Updated. PTAL..